src/openforms/registrations/contrib/objects_api/tests/test_json_schema.py

```diff
@@ -163,6 +165,12 @@ class IterJsonSchemaTests(SimpleTestCase):
     simply be tested as ``([], CURRENT_SCHEMA_TESTED)``.
     """
 
+    def test_no_spec_default(self):
+        try:
+            list(iter_json_schema_paths(JSON_SCHEMA_NO_SPEC))
+        except Exception as e:
+            self.fail(f"Unexpected exception: {e}")
```

Comment:
```suggestion
        except Exception as e:
            raise self.failureException("It broke") from e
```
I don't like traceback information being butchered by stringifying it :)